src/libraries/Microsoft.Extensions.Configuration.Abstractions/src/ConfigurationExtensions.cs

```diff
@@ -73,6 +73,8 @@ public static string GetConnectionString(this IConfiguration configuration, stri
         /// <summary>
         /// Determines whether the section has a <see cref="IConfigurationSection.Value"/> or has children
         /// </summary>
+        /// <param name="section">The <see cref="IConfigurationSection"/> to enumerate.</param>
+        /// <returns>A flag to indicate whether the section has values or not.</returns>
```

Comment:
```suggestion
        /// <returns><see langword="true" /> if the section has values or children; otherwise, <see langword="false" />.</returns>
```